pkgs/development/python-modules/pixelmatch/default.nix

```diff
@@ -0,0 +1,31 @@
+{ lib, buildPythonPackage, fetchgit, pytestCheckHook, pillow, twine, poetry }:
+
+buildPythonPackage rec {
+  pname = "pixelmatch";
+  version = "0.2.1";
+  format = "pyproject";
+
+  # test fixtures are stored in LFS
+  src = fetchgit {
+    url = "https://github.com/whtsky/pixelmatch-py";
+    rev = "v${version}";
+    sha256 = "1lb80lz6pi014hf7a49fys1b3j89jz4i8ghpjk83fqp0sf25w2p9";
+    fetchLFS = true;
+  };
+
+  nativeBuildInputs = [
+    poetry
```

Comment:
Let see if upstream is willing to move to  `poetry-core`.

https://github.com/whtsky/pixelmatch-py/pull/81

Reply:
[0.2.2](https://github.com/whtsky/pixelmatch-py/releases/tag/v0.2.2) was release.